src/actions/index.js

```diff
@@ -8,7 +8,7 @@ export const removeBook = (book) => ({
   payload: book,
 });
 
-export const changeFilter = (filt) => ({
-  type: 'Change',
+export const CHANGE_FILTER = (filt) => ({
```

Comment:
### [Optional]

For good naming conversion, it would be nice to use `filter` instead of `filt`